src/pages/ListPets/AdoptionForm/index.tsx

```
@@ -37,6 +42,49 @@ const formSchema = z.object({
+
+      try {
+        setLoading(true);
+        // Fetch pet by slug - this is correct now as we're passing a slug
+        const response = await axiosClient.get(
+          `${API_ENDPOINT.PET.DETAIL.replace(":slug", slug)}`
+        );
+
+        // Process the response to extract pet data
+        let petData;
+        if (response.data && response.data.data) {
+          petData = response.data.data;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)